### PR TITLE
Remove use of Enzyme from the Image container

### PR DESCRIPTION
**Pull request by FK78**
Part of #8458

**Overall change:**
_Remove use of Enzyme form the Image container._

**Code changes:**

- _Use `render` from `@testing-library/react` instead of `enzyme`._
- _Refactor test to check that the `noscript` tag is rendering while the `img` tag is not._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Image/index.test.jsx [hunk 1]
```diff
@@ -87,9 +88,12 @@ describe('Image', () => {
     );
 
     it('should render a lazyload container instead of an image if the image is after the 3rd block', () => {
-      // Render using enzyme to capture noscript contents
-      const container = render(<ImageContainer position={[4]} {...data} />);
-      expect(container).toMatchSnapshot();
```
Comment (jroebu14) [inline src/app/containers/Image/index.test.jsx:92]: woops did I delete this snapshot?! maybe add this back in

Comment (simonsinclair) [reply src/app/containers/Image/index.test.jsx:92]: 😱

Comment (simonsinclair) [reply src/app/containers/Image/index.test.jsx:92]: With the additional assertions, it can probably remain deleted? 😄

Comment (FK78) [reply src/app/containers/Image/index.test.jsx:92]: I actually removed it by accident 🙈 

Added back in now.

**Review by simonsinclair — APPROVED** [review]

**Review by joshcoventry — APPROVED** [review]

**Pull request merged**